README.md

```diff
@@ -170,8 +170,10 @@ Read more in the [components docs](docs/components.md).
 
 These third-party libraries are great for use with mdx-deck.
 
+- [mdx-code][]: Runnable code playgrounds for MDX Deck.
```

Comment:
Thanks! Could you put this after CodeSurfer since it was added first?

Reply:
Yup!